package.json

```diff
@@ -38,6 +38,7 @@
     "semver": "^4.3.3",
     "shelljs": "^0.3.0",
     "through2": "^0.6.3",
+    "typescript": "1.8.0",
```

Comment:
This is a developer dependency. This pins the build to a known version so our CI doesn't break.